components/brave_webtorrent/browser/BUILD.gn

```diff
@@ -13,11 +15,17 @@ source_set("browser") {
     "//components/pref_registry",
     "//components/prefs",
     "//content/public/browser",
-    "//extensions/browser",
-    "//extensions/common",
+    "//extensions/buildflags",
     "//net",
   ]
 
+  if (enable_extensions) {
```

Comment:
so actually looking more closely at this, we should remove this guard because it doesn't exist in the source. However, this BUILD.gn file should have `assert(enable_extensions)` at the top because webtorrent currently depends on extensions

Reply:
ok, I removed the guard and add assert(enable_extensions) at the top of the file. Thanks.